src/redux/books/books.js

```diff
@@ -0,0 +1,23 @@
+// Actions
+const ADD = 'bookstore/book/ADD';
+const REMOVE = 'bookstore/book/remove';
```

Comment:
Defines `ADD_BOOK` and `REMOVE_BOOK` as action types ✔

Reply:
Please be sure to follow the naming conventions on your `REMOVE` action as well 🤓 If you want a bit more information on why to follow these patterns and conventions check the following [link](https://decembersoft.com/posts/a-simple-naming-convention-for-action-creators-in-redux-js/) 📝